src/Frontend/config/initialState.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * This file is part of the Sententiaregum project.
+ *
+ * (c) Maximilian Bosch <maximilian.bosch.27@gmail.com>
+ * (c) Ben Bieler <benjaminbieler2014@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+export default {
+  menu: {
+    items: []
+  },
+  user: [],
+  locales: {
```

Comment:
The state should contain the current locale, too.

```
{
  available: { 'de': 'Deutsch', ... },
  current: 'en'
}
```

The other code can be kept for now (so the state tree should be changed, but no reducer should manage the current locale for now). The security system is unfortunately way too coupled with the locale (since it's user-specific) and I'll try to refactor all of that when working on the security system.